projects/plugins/jetpack/_inc/client/traffic/site-stats.jsx

```diff
@@ -256,8 +256,9 @@ class SiteStatsComponent extends React.Component {
 								>
```

Comment:
This suggestion slightly bloats this task's scope, but could you move the Odyssey toggle above the `48-hour traffic snapshot` toggle?